test/test_docstring_detection.py

```diff
@@ -0,0 +1,44 @@
+import tokenize
+from unittest import TestCase
+
+from flake8_quotes import Token, get_docstring_tokens
+from test.test_checks import get_absolute_path
+
+
+class GetDocstringTokensTests(TestCase):
+    def _get_docstring_tokens(self, filename):
+        with open(get_absolute_path(filename), 'r') as f:
+            tokens = [Token(t) for t in tokenize.generate_tokens(f.readline)]
+        return get_docstring_tokens(tokens)
+
+    def test_get_docstring_tokens_absent(self):
```

Comment:
I think we'll be fine with separating the files -- these should be considered unit tests for now. If we want to find/test more complex cases, then we can add them as more files and search for them via fuzzing.

I'm not focusing on indentation, I'm focused on covering the base cases directly as well as testing any potential edge cases. This is regardless of the current implementation as we want tests to work even if we switch the underlying setup. It just so happens though that Python mostly fluctuates around indentation when altering how functions/classes are defined